model/src/main/java/org/projectnessie/model/ContentResponse.java

```diff
@@ -26,6 +26,7 @@
 @Value.Immutable
 @JsonSerialize(as = ImmutableContentResponse.class)
 @JsonDeserialize(as = ImmutableContentResponse.class)
+@JsonView(Views.V2.class)
```

Comment:
Does it work at class level?.. I'm not sure.

Reply:
My preference would actually be to remove all views annotations from this class because it is not referenced by v1 API methods.

Reply:
Does it matter?

Reply:
To avoid confusion, I guess. I would not want us to have an annotation that is not effective.